res/controllers/Denon-MC7000-scripts.js

```diff
@@ -639,8 +692,9 @@ MC7000.pitchFaderPosition = function(channel, control, value, status, group) {
 
 // Next Rate range toggle
 MC7000.nextRateRange = function(midichan, control, value, status, group) {
-    if (value === 0)
+    if (value === 0) {
```

Comment:
```suggestion
    if (!value) {
```

and elsewhere.

Reply:
why? ... there shall be no trigger when the value is 0 (release of button)

Reply:
I don't understand. The expression evaluates to true iff value is equal to 0.

Reply:
Yes - both are working and neither is incorrect. But excuse me, "value === 0" is much more intuitive to me. I will not change it because of the personal taste of a reviewer. Next one will say differently again. Thanks for understanding.